### PR TITLE
Add handler for _ah/start

### PR DESCRIPTION
**Pull request by achuinard**
Required for App Engine

### CODE REVIEW

**Comment by renaudcerrato** [comment]
Thanks! Can you provide a link to the documentation about that requirements?

File: src/main/java/repo/resource/RepositoryResource.java [hunk 1]
```diff
@@ -37,6 +37,12 @@
 
     private final GcsService gcs = GcsServiceFactory.createGcsService();
     private final BlobstoreService blobstore = BlobstoreServiceFactory.getBlobstoreService();
+    
+    @GET
+    @Path("/_ah/start")
+    public Boolean startup() {
+        return Boolean.TRUE;
```
Comment (renaudcerrato) [inline src/main/java/repo/resource/RepositoryResource.java]: `return Response.accepted().build();` is cleaner.

File: src/main/java/repo/resource/RepositoryResource.java [hunk 2]
```diff
@@ -37,6 +37,12 @@
 
     private final GcsService gcs = GcsServiceFactory.createGcsService();
     private final BlobstoreService blobstore = BlobstoreServiceFactory.getBlobstoreService();
+    
+    @GET
+    @Path("/_ah/start")
+    public Boolean startup() {
```
Comment (renaudcerrato) [inline src/main/java/repo/resource/RepositoryResource.java]: `public Response startup() {` is cleaner.

**Comment by renaudcerrato** [comment]
Thanks for the PR. I just added my two-cents comment, could you resolve them before I merge? Thanks again!

**Comment by achuinard** [comment]
Good feedback! Updated the PR, thanks!

**Pull request merged**